### PR TITLE
#14 done add footer

### LINKED ISSUE #14: add footer

### PR DESCRIPTION
**Pull request by toanmu10**
close #14

### CODE REVIEW

**Review by Manhisme09 — COMMENTED** [review]
LGTM

**Review by Manhisme09 — COMMENTED** [review]
LGTM

**Comment by ngoctaitrieu** [comment]
LGTM

File: resources/js/slick.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $('.slider').slick({
     slidesToShow: 2,
```
Comment (phamthimylinhhaui) [inline resources/js/slick.js:2]: js 2 space

**Review by phamthimylinhhaui — COMMENTED** [review]
LGMT

**Review by hapo-nghialuu — APPROVED** [review]
LGTM

**Pull request merged**